vertx-sockjs-service-proxy/src/test/resources/vertx-js/vertx-eventbus.js

```diff
@@ -308,6 +340,16 @@
     }
   };
 
+  EventBus.prototype.reconnectEnabled = function (enable) {
```

Comment:
I was thinking the same when I wrote this, but there's already an exposed method called pingEnabled which I thought to match.